clang/lib/Sema/SemaChecking.cpp

```diff
@@ -5246,6 +5246,23 @@ bool CheckVectorElementCallArgs(Sema *S, CallExpr *TheCall) {
   return true;
 }
 
+bool CheckAllArgsAreFloatRepresentation(Sema *S, CallExpr *TheCall) {
```

Comment:
Consider changing the function name to either
`CheckAllArgsAreFloatType`, `CheckAllArgsAreFloats` or `CheckAllArgsHaveFloatRepresentation`